src/Analyzer.PowerShellRuleEngine.UnitTests/PowerShellRuleEngineTests.cs

```diff
@@ -69,32 +82,89 @@ public void AnalyzeTemplate_ValidTemplate_ReturnsExpectedEvaluations(string temp
 
             Assert.AreEqual(expectedErrorCount, failedEvaluations.Count);
 
-            Assert.AreEqual(failedEvaluations.Count, lineNumbers.Length);
-            for (int errorNumber = 0; errorNumber < lineNumbers.Length; errorNumber++)
+            // PSRule evaluations can change order depending on the OS:
+            foreach(var expectedLineNumber in expectedLineNumbers)
             {
-                Assert.AreEqual(lineNumbers[errorNumber], failedEvaluations[errorNumber].Result.LineNumber);
+                var matchingEvaluation = failedEvaluations.Find(evaluation => evaluation.Result.LineNumber == expectedLineNumber);
+                failedEvaluations.Remove(matchingEvaluation);
+            }
+            Assert.IsTrue(failedEvaluations.IsEmptyEnumerable());
+        }
+
+        [DataTestMethod]
+        [DataRow(true, DisplayName = "Repeated rules are excluded when running all the rules")]
+        [DataRow(true, DisplayName = "Repeated rules are excluded when running only the security rules")]
+        public void AnalyzeTemplate_ValidTemplate_ExcludesRepeatedRules(bool runsAllRules)
+        {
+            var templateFilePath = Path.Combine(templatesFolder, "excluded_rules.json");
```

Comment:
Can you rename the file to TestExcludedRules.json?

Reply:
I thought it was a rule file or configuration file and it confused me.

Reply:
Do you think it's necessary even considering it is in the "templates" folder?

Reply:
If I plan to make a change to this test, it could help me save time understanding what it going on. And if I want to use this file in a test, would also help. 
If there is a different name, could also be good.